packages/language-server/src/prisma-fmt/lint.ts

```diff
@@ -11,16 +12,13 @@ export default function lint(text: string, onError?: (errorMessage: string) => v
   console.log('running lint() from prisma-fmt')
   try {
     const result = prismaFmt.lint(text)
-    return JSON.parse(result) // eslint-disable-line @typescript-eslint/no-unsafe-return
-  } catch (errors) {
-    const errorMessage = "prisma-fmt error'd during linting.\n"
 
-    if (onError) {
-      onError(errorMessage + errors) // eslint-disable-line @typescript-eslint/restrict-plus-operands
-    }
+    return JSON.parse(result) as LinterError[]
+  } catch (e) {
+    const err = e as Error
+
+    handleWasmError(err, 'lint', onError)
```

Comment:
Intentionally different name than other uses?

Reply:
+1

Reply:
Also not sure here 😅